src/config/selectors.js

```diff
@@ -161,3 +161,9 @@ export const CROP_MODAL_CONFIRM_BUTTON_CLASSNAME = 'cropModalConfirmButton';
 export const MEMBER_PROFILE_AVATAR_UPLOAD_BUTTON_CLASSNAME =
   'memberProfileAvatarUploadButton';
 export const ZIP_DASHBOARD_UPLOADER_ID = 'zipDashboardUploader';
+
+export const buildCustomizedTagsSelector = (index) =>
+  `customizedTagsPreview-${index}`;
+
+export const CATEGORIES_SELECTION_VALUE_SELECTOR =
+  'div.MuiChip-root.MuiAutocomplete-tag.MuiChip-deletable > span';
```

Comment:
It's still not really precise, it will fail if there's two autocomplete components in the same page. You need to specify an id which wraps the component you want to select to avoid the problem I've mentioned.